Jenkinsfile

```diff
@@ -624,6 +629,9 @@ pipeline {
                             if (BRANCH_NAME == MASTER_BRANCH) {
                                 echo "publishing next to $TEST_NPM_REGISTRY"
```

Comment:
I suggest using a different tag for master for now otherwise we'll end up deploying both to the Artifactory with potentially different levels of code

Reply:
Good point! thanks